sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -205,15 +187,15 @@ public partial class EntityDataSource
         public string EntityId { get { throw null; } }
         public string Name { get { throw null; } }
     }
-    public partial class EntityRecognitionPiiTasksItem : Azure.AI.TextAnalytics.TaskState
+    public partial class ExtractKeyPhrasesActionResult : Azure.AI.TextAnalytics.TextAnalyticsActionDetails
     {
-        internal EntityRecognitionPiiTasksItem() { }
-        public Azure.AI.TextAnalytics.RecognizePiiEntitiesResultCollection Results { get { throw null; } }
+        internal ExtractKeyPhrasesActionResult() { }
+        public Azure.AI.TextAnalytics.ExtractKeyPhrasesResultCollection Result { get { throw null; } }
     }
-    public partial class EntityRecognitionTasksItem : Azure.AI.TextAnalytics.TaskState
+    public partial class ExtractKeyPhrasesOptions : Azure.AI.TextAnalytics.TextAnalyticsRequestOptions
     {
-        internal EntityRecognitionTasksItem() { }
-        public Azure.AI.TextAnalytics.RecognizeEntitiesResultCollection Results { get { throw null; } }
+        public ExtractKeyPhrasesOptions() { }
+        public new string ModelVersion { get { throw null; } set { } }
```

Comment:
I missed this in previous iterations.  `TextAnalyticsRequestOptions` already has `ModelVersion` so it shouldn't be declared here again. I believe that is why you had to do `new string`.
Same for the other `xxOptions`